build.sh

```diff
@@ -3,18 +3,22 @@ uname_string=`uname | sed 'y/LINUXDARWINFREEOPENPCBSDMSYS/linuxdarwinfreeopenpcb
 host_arch=`uname -m | sed 'y/XI/xi/'`
 case "$uname_string" in
   *linux*)
+    BUILD_SHARED_LIBS=OFF
     HOST_NATIVE="$host_arch"-linux-gnu
     JOBS="-j`grep ^processor /proc/cpuinfo|wc -l`"
     ;;
   *freebsd*)
+    BUILD_SHARED_LIBS=OFF
     HOST_NATIVE="$host_arch"-freebsd
     JOBS="-j`sysctl kern.smp.cpus | sed 's/kern.smp.cpus: //'`"
     ;;
   *darwin*)
+    BUILD_SHARED_LIBS=OFF
     HOST_NATIVE=x86_64-apple-darwin10
     JOBS="-j1"
     ;;
   *msys*)
+    BUILD_SHARED_LIBS=ON
```

Comment:
Because Windows could use shared libraries in the same directory by default.
[Dynamic-Link Library Search Order (Windows) Standard Search Order for Desktop Applications](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682586(v=vs.85).aspx#standard_search_order_for_desktop_applications)